src/frontend/screens/Settings/components/EnableNtsync.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ToggleSwitch } from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+import ContextProvider from 'frontend/state/ContextProvider'
+import SettingsContext from '../SettingsContext'
+import InfoIcon from 'frontend/components/UI/InfoIcon'
+
+const EnableNtsync = () => {
```

Comment:
If this should work only with Proton then we should hide it for other wine versions.
Check this line to see how we do it:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/9a9c1568e93887ea052e8843d3f5da8fa9f3a909/src/frontend/screens/Settings/components/AutoDXVK.tsx#L23